netsim/devices/eos.py

```diff
@@ -65,6 +66,32 @@ def check_dhcp_clients(node: Box, topology: Box) -> None:
       category=log.IncorrectType,
       module='quirks')
 
+def configure_ceos_attributes(node: Box, topology: Box) -> None:
+  serialnumber = node.eos.get('serialnumber',None)
+  systemmacaddr = node.eos.get('systemmacaddr',None)
+  if not serialnumber and not systemmacaddr:
+    return
+
+  if 'clab' not in node or node.clab.kind != "ceos":
+    log.error(
+      f"eos.serialnumber and eos.systemmacaddr can only be set for Arista cEOS containers (node {node.name}).",
+      category=Warning,
+      more_hints=['Use libvirt.uuid to influence serial number on EOS virtual machines'],
+      module='quirks')
+    return
+  
+  mnt_config = '/mnt/flash/ceos-config'
+  for ct in node.get('clab.binds',[]):
+    if mnt_config in ct:
+      log.error(
+        f"{mnt_config} file is already mapped, unable to configure eos.serialnumber (node {node.name}).",
+        category=log.Skipped,
+        module='quirks')
+      return
+
+  data.append_to_list(node.clab,'binds',f'clab_files/{node.name}/ceos-config:{mnt_config}')
```

Comment:
I think this happens automatically after the config_template is rendered successfully

Reply:
I took the pragmatic approach of creating a fake template, dumping the node data in the quirks, and recreating the relevant data structures.